ui_framework/doc_site/src/views/collapse_button/collapse_button_aria.js

```diff
@@ -0,0 +1,47 @@
+import React, { Component } from 'react';
+
+import {
+  KuiCollapseButton
+} from '../../../../components';
+
+import { htmlIdGenerator } from '../../../../services';
+
+export default class extends Component {
+
+  constructor(props) {
+    super(props);
+    this.state = {
+      isExpanded: false
+    };
+  }
+
+  onToggleContent = (ev) => {
+    ev.preventDefault();
+    this.setState((state) => ({
+      isExpanded: !state.isExpanded
+    }));
+  }
+
+  render() {
+    const { isExpanded } = this.state;
+    const idGen = htmlIdGenerator();
+    return (
+      <div>
+        <KuiCollapseButton
+          onClick={this.onToggleContent}
+          direction={isExpanded ? 'down' : 'up'}
+          aria-label="Toggle panel"
+          aria-expanded={isExpanded}
+          aria-controls={idGen('collapsable')}
+        />
+        <div
+          id={idGen('collapsable')}
+          style={{ display: isExpanded ? 'block' : 'none' }}
+        >
+          Here is some collapsable content.
```

Comment:
Typo: I think this should be "collapsible". We should probably update the generated IDs with this spelling too.